modules/etcd-increase-db.adoc

```diff
@@ -0,0 +1,185 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="etcd-increase-db_{context}"]
+= Increasing the database size for etcd
+
+You can set the disk quota in gigabytes for each etcd instance. If you set a disk quota for your etcd instance, you can specify integer values from 8 to 32. The default value is 8. You can specify only increasing values.
+
```

Comment:
Could we add something along the lines of:
Increasing this value does not immediately reserve this amount of disk space, but instead allows etcd to grow up to that value if it needs to. Please ensure that etcd is running on a dedicated disk larger than the value you wish to set.

(Not sure if we need to be more specific about how much larger the disk needs to be)

Reply:
Done. See line 15.